tests/integration/distributed/test_simple_distributed/test_integration.py

```diff
@@ -0,0 +1,50 @@
+from pathlib import Path
+import pytest
+import sys
+import time
+
+from tests.helpers import (
+    start_docker_compose,
+    stop_docker_compose,
+    send_flow,
+    call_api,
+    get_results,
+)
+
+directory = Path('tests/integration/distributed/test_simple_distributed/')
+compose_yml = directory / 'docker-compose.yml'
+flow_yml = directory / 'flow.yml'
+pod_dir = directory / 'pods'
+
+
+def test_flow():
+    if Path.cwd().name != 'jinad':
+        sys.exit(
+            'test_simple_distributed.py should only be run from the jinad base directory'
+        )
+
+    start_docker_compose(compose_yml)
```

Comment:
we could use `try finally` in every test to ensure `docker compose` is called even with failure, otherwise a failure will make everything fail